### PR TITLE
Postgresql 9.2 (fix styles)

### PR DESCRIPTION
**Pull request by sigerello**
Fixed styles as in #602

### CODE REVIEW

File: postgresql92.rb [hunk 1]
```diff
@@ -1,29 +1,29 @@
-# -*- coding: utf-8 -*-
-require 'formula'
-
 class Postgresql92 < Formula
-  homepage 'http://www.postgresql.org/'
-  url 'http://ftp.postgresql.org/pub/source/v9.2.8/postgresql-9.2.8.tar.bz2'
-  sha256 '568ba482340219097475cce9ab744766889692ee7c9df886563e8292d66ed87c'
-  revision 1
```
Comment (jacknagel) [inline postgresql92.rb]: please don't remove the revision, it was added on purpose in 4aee0cf846733c18986a3493e66ec257f25eb8e3.

File: postgresql92.rb [hunk 2]
```diff
@@ -170,7 +173,7 @@ def plist; <<-EOS.undent
 -override python_libspec = -framework Python
 -override python_additional_libs =
  endif
- 
+
```
Comment (jacknagel) [inline postgresql92.rb]: don't change whitespace inside patches.

**Comment by jacknagel** [comment]
comments apply to the other PRs as well

**Comment by sigerello** [comment]
@jacknagel Fixed in all PRs

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
Can you steal the [tcl block in Homebrew/Homebrew's Postgres](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/postgresql.rb#L59-L67) formula to fix the Mountain Lion TCL issues on these formula? Thanks!

**Comment by sigerello** [comment]
@DomT4 Done. All is well.

**Comment by DomT4** [comment]
Merged! Thanks for your contribution to Homebrew, we appreciate it!

**Pull request closed without merging**